src/pages/messaging/references/default-properties/index.mdx

```diff
@@ -17,16 +17,237 @@ If you need to brush up on properties, [read more](/management/configuration-api
+| **statistics**    | [`first_chat_date`](#first_chat_date) [`website_last_activity`](#website_last_activity)                                                                                                                                                                                                                                                                                                                                                                          |
+| **status**        | [`delivered`](#delivered)                                                                                                                                                                                                                                                                                                                                                                                                                                        |
+| **test**          | [`bool_property`](#bool_property) [`int_property`](#int_property) [`string_property`](#string_property) [`tokenized_string_property`](#tokenized_string_property)                                                                                                                                                                                                                                                                                                |
+| **translation**   | [`source_lang_code`](#source_lang_code) [`target_lang_code`](#target_lang_code) [`target_message`](#target_message)                                                                                                                                                                                                                                                                                                                                              |
+| **url_details**   | [`description`](#description) [`image_height`](#image_height) [`image_width`](#image_width) [`image_url`](#image_url) [`title`](#title) [`url`](#url)                                                                                                                                                                                                                                                                                                            |
+
+<Section>
+<Text>
+
+## chats
+
+### queue.routing_priority
+
+Specifies routing priority for queued chats in the group.
+
+|               |                          |
+| ------------- | ------------------------ |
+| **Access**    | `rw` for Agents in group |
+| **Data type** | `int`                    |
```

Comment:
yeah, not sure about that, but if @kamilatokarska is ok with this, then I'm not going to press you on the matter 😉

Reply:
I think I would go that I'm okay with it - it is visible from the access. In case some questions start appear, I would make it more clear, but for now, I would leave it how it is :)